server/modules/analysis/service/rChainService/rChain.js

```diff
@@ -144,6 +151,10 @@ class RChain {
     this._fileLogin = await new RFileLogin(this).init()
     this._fileReadData = await new RFileReadData(this).init()
 
+    // Init categories
+    this._listCategories = new ListCategories(this)
+    await this._fileReadData.appendContent(...this._listCategories.scripts)
```

Comment:
Add this to ```RFileReadData.init()```

Reply:
what about DFResults? Should it access listCategories with rChain.fileReadData.listCategories then? Or should RChain expose listCategories from the instance variable _fileReadData?